run/GCHP/createRunDir.sh

```diff
@@ -474,50 +488,44 @@ while [ "${valid_met}" -eq 0 ]; do
 	
 	# Set text files containing settings for met data. Different settings based options aboves.
 	if [[ ${met_file_type} = "processed_ll" ]]; then
-	    RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/geosit.preprocessed_ll.txt)\n"
+	    RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/geosit.preprocessed_0.5x0.625.txt)\n"
 	
 	elif [[ ${met_file_type} = "processed_cs" ]]; then
-	    if [[ ${adv_flux_src} = "mass_flux" ]]; then
-	        RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/geosit.preprocessed_mass_flux.txt)\n"
-	    elif [[ ${adv_flux_src} == "wind" ]]; then
-		RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/geosit.preprocessed_wind_cs.txt)\n"
+
+	    if [[ ${adv_flux_src} = "1hr_mass_flux" ]]; then
+	        RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/advection_met/geosit.preprocessed_1hr_c180_mass_flux.txt)\n"
+	    elif [[ ${adv_flux_src} == "3hr_wind" ]]; then
+		RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/advection_met/geosit.preprocessed_3hr_c180_wind.txt)\n"
 	    fi
-	    RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/geosit.preprocessed_cs.txt)\n"
+	    RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/geosit.preprocessed_c180.txt)\n"
+
 	else
+
+	    # Using raw GEOS-IT data downloaded from GMAO
 	    if [[ ${use_discover} = "y" ]]; then
-		
-		# Settings for advection vars in ExtData.rc, running on discover
-	        if [[ ${adv_flux_src} = "mass_flux" ]]; then
-	            RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/discover/geosit.raw_mass_flux.discover.txt)\n"
-	        elif [[ ${adv_flux_src} == "wind" && ${met_file_type} == "raw_cs" ]]; then
-	            RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/discover/geosit.raw_wind_cs.discover.txt)\n"  
-	        elif [[ ${adv_flux_src} == "wind" && ${met_file_type} == "raw_ll" ]]; then
-	            RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/discover/geosit.raw_wind_ll.discover.txt)\n"
-		fi
 
-		# Settings for all other met vars in ExtData.rc, running on discover
-		if [[ ${met_file_type} = "raw_cs" ]]; then
-		    RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/discover/geosit.raw_cs.discover.txt)\n"
-		elif [[ ${met_file_type} = "raw_ll" ]]; then
-		    RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/discover/geosit.raw_ll.discover.txt)\n"
+		if [[ ${met_file_type} = "raw_ll" ]]; then
+		    RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/discover/geosit.raw_0.5x0.625.txt)\n"
+		else
+	            if [[ ${adv_flux_src} = "1hr_mass_flux" ]]; then
+			RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/discover/geosit.raw_1hr_c180_mass_flux.txt)\n"
+	            elif [[ ${adv_flux_src} = "3hr_wind" ]]; then
+			RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/discover/geosit.raw_3hr_c180_wind.txt)\n"
+		    fi
+		    RUNDIR_VARS+="$(cat ${metSettingsDir}/geosit/discover/geosit.raw_c180.txt)\n"
 		fi
 
 	    elif [[ ${use_discover} = "n" ]]; then
```

Comment:
Can `elif [[ ${use_discover} = "n" ]]; then` just be an `else` here?

Reply:
Yes, and I just changed it to use else, with a comment that the block is for not discover. I think I didn't put else before just to be explicit about what the else was referring to.